services/web/server/src/simcore_service_webserver/products/_middlewares.py

```diff
@@ -70,7 +70,9 @@ async def discover_product_middleware(request: web.Request, handler: Handler):
         request[RQ_PRODUCT_KEY] = product_name
 
     assert (  # nosec
-        request.get(RQ_PRODUCT_KEY) is not None or request.path == "/socket.io/"
+        request.get(RQ_PRODUCT_KEY) is not None
+        or request.path == "/socket.io/"
+        or request.path.startswith("/dev/doc")
```

Comment:
@pcrespov note this change. Is it OK for you? I had to do it in order to make the swagger documentation available.